installers/dhall/demo.dhall

```diff
@@ -6,5 +6,5 @@
 , installDirectorySuffix = " Demo"
 , macPackageSuffix       = "Demo"
 , walletPort             = 8092
-, extraNodeArgs          = [ "--metrics", "--ekg-server", "localhost:8085", "+RTS", "-T", "-RTS" ] : List Text
+, extraNodeArgs          = [ "--network", "local" ] : List Text
```

Comment:
we may need 2 sets of node args in the new version
one for the node, and one for the wallet

Reply:
Yeh agree. At the moment we are just firing up the `cardano-wallet-launcher` which manages both the node and wallet, but this will change as soon as we move to Jormangandr